src/Integration.UnitTests/Connection/ConnectionWorkflowTests.cs

```diff
@@ -185,6 +185,8 @@ public void ConnectionWorkflow_DownloadServiceParameters_RegexPropertyNotSet_Set
             var expectedExpression = ServerProperty.TestProjectRegexDefaultValue;
 
             ConnectionWorkflow testSubject = new ConnectionWorkflow(this.host, new RelayCommand(AssertIfCalled));
+            var connectionInfo = new ConnectionInformation(new Uri("http://server"));
+            testSubject.ConnectedServer = connectionInfo;
```

Comment:
Might be worth extracting these three lines into a helper. They're duplicated in several locations.